snmp/datadog_checks/snmp/cmd.py

```diff
@@ -0,0 +1,134 @@
+# (C) Datadog, Inc. 2020-present
```

Comment:
Any links to the low-level API documentation we can put in here for reference, or is this API not documented and this was built by reading through the source code? Right now I can see several similarities with the "high-level" (?) `hlapi` command functions (esp with `GET NEXT`), but I'm curious where this all came from.

Reply:
It was all implemented looking at the source code.  This a mix of https://github.com/etingof/pysnmp/blob/master/pysnmp/hlapi/v3arch/asyncore/sync/cmdgen.py and https://github.com/etingof/pysnmp/blob/master/pysnmp/hlapi/v3arch/asyncore/cmdgen.py code extracted and simplified for our use case.